### ISSUE TITLE
Git clone/pull doesn't clone all files [Windows]

### ISSUE THREAD

**Issue by clay53** [GM-Coding-Club/birdhouse#3]
When doing git pull or git clone it doesn't copy all the folders/files and the clone is missing git functionality.

**Comment by clay53** [comment]
Warn:
```
warning: unable to access 'node_modules/npm/node_modules/libnpmhook/node_modules/npm-registry-fetch/node_modules/make-fetch-happen/node_modules/http-proxy-agent/node_modules/agent-base/node_modules/es6-promisify/node_modules/es6-promise/lib/es6-promise/.gitattributes': Filename too long
fatal: cannot create directory at 'node_modules/npm/node_modules/libnpmhook/node_modules/npm-registry-fetch/node_modules/make-fetch-happen/node_modules/http-proxy-agent/node_modules/agent-base/node_modules/es6-promisify/node_modules/es6-promise/lib/es6-promise/promise': Filename too long
```

**Comment by clay53** [comment]
This was due to `node_modules/` being included in the files which I didn't know was bad at the time. This was fixed really early on in 5e2c7d06f0199fab79e82d67350e3afd391468b6
